test/evergreen.yml

```diff
@@ -141,7 +141,7 @@ functions:
           # Use the Windows powershell script to configure the CMake build.
           # We execute it in a powershell environment as its easier to detect and source the Visual Studio
           # toolchain in a native Windows environment. We can't easily execute the build in a cygwin environment.
-          powershell.exe  -NonInteractive '.\test\evergreen\build_windows.ps1' -configure 1 ${windows_configure_flags|}
+          ${configure_env_vars|} powershell.exe  -NonInteractive '.\test\evergreen\build_windows.ps1' -configure 1 ${windows_configure_flags|}
```

Comment:
`configure_env_vars` and `compile_env_vars` weren't present on 6.0 previously. This backport introduces them as I don't expect to backport the original commit